media-libs/gst-plugins-good/files/gst-plugins-good-1.20.0-lame-feature-option.patch

```diff
@@ -0,0 +1,23 @@
+commit d03971dac7b32a6ffcbf161853e017f65ae7c22f
```

Comment:
If possible, link to the exact commit too, to ease checking of things in future, just pop a link at the top of the patch. But don't bother doing it now given the link to the MR is already there :)

Reply:
For gitlab MR link is more funcitonal since it usually provides more context (and also these days in gstreamer, where the change is going to be backported)